packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -209,6 +209,11 @@ export interface ExpoConfig {
      * Array of glob patterns specifying which files should be included in updates. Glob patterns are relative to the project root. A value of `['**']` will match all asset files within the project root. When not supplied all asset files will be included. Example: Given a value of `['app/images/** /*.png', 'app/fonts/** /*.woff']` all `.png` files in all subdirectories of `app/images` and all `.woff` files in all subdirectories of `app/fonts` will be included in updates.
      */
     assetPatternsToBeBundled?: string[];
+    /**
+     * Allow expo-updates to be run in danger mode. This may allow for the app to be bricked if an update fails.
+     * @default false
+     */
+    disableAntiBrickingMeasures?: boolean;
```

Comment:
@Kudo could we put this in the `extra` section, since this is temporary? https://docs.expo.dev/versions/latest/config/app/#extra

Reply:
I don't think it's temporary, unfortunately :( Seems like features like this likely will be the new direction of the library.

I'm adding to the xdl-schemas now and will add you as a reviewer.